packages/block-editor/src/components/block-heading-level-dropdown/heading-level-icon.js

```diff
@@ -39,5 +40,9 @@ const LEVEL_TO_PATH = {
  * @return {?WPComponent} The icon.
  */
 export default function HeadingLevelIcon( { level } ) {
-	return LEVEL_TO_PATH[ level ] ?? null;
+	if ( LEVEL_TO_PATH[ level ] ) {
+		return <Icon icon={ LEVEL_TO_PATH[ level ] } />;
+	}
```

Comment:
This fixes the invisible icons for the mobile editor. 

From testing the web editor, this change did not appear to cause issues. @carolinan would you please confirm this is appropriate? Thanks! 🙇🏻

Reply:
It works as expected, thank you.